install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -271,7 +279,9 @@ spec:
               # If available, SYS_ADMIN can be removed.
               #- PERFMON
               #- BPF
-          {{- end}}
+            drop:
+              - ALL
+          {{- end }}
```

Comment:
For reviewers, drop all capabilities because we don't want to rely on default capabilities set by container runtime.
Currently the default is different between [cantainerd](https://github.com/moby/moby/blob/8abb84c7342a19009d777ef3d871c0b2a5c032a5/oci/caps/defaults.go#L6-L19) and [cri-o](https://github.com/cri-o/cri-o/blob/a12d4de4666875483659f158f5047185c519a40e/internal/config/capabilities/capabilities.go#L14-L27) and it can be configured or be change in the future by vendor.